kubernetes/README.md

```diff
@@ -210,33 +210,34 @@ If you regenenerate credentials for your SCIM bridge:
    ```
 
    Kubernetes automatically updates the credentials file mounted in the Pod with the new Secret value.
+
 3. Test your SCIM bridge using the new bearer token associated with the regenerated `scimsession` file.
 4. Update your identity provider configuration with the new bearer token.
 
 ## Appendix: Resource recommendations
 
 The default resource recommendations for the SCIM bridge and Redis deployments are acceptable in most scenarios, but they may fall short in high-volume deployments where a large number of users and/or groups are being managed. We strongly recommend increasing the resources for both the SCIM bridge and Redis deployments.
```

Comment:
Since the performance improvements we've made (see https://docs.google.com/document/d/1cD5GY8kNm-RwqqA1sklwzx77yoknmGV-bfaTEvUv9sg/edit), we no longer need to recommend customers bump resources for their Redis deployments. So this statement can read:

> We strongly recommend increasing the resources for the SCIM bridge deployment.

I would also like to add a comment about deploy resources vs. normal everyday use of the SCIM bridge after this comment. Something like:

> Once all users and groups have been provisioned, it is safe to lower these resources back to their defaults for everyday use of the SCIM bridge.

Reply:
Thanks. Thats good callout. <br/>
For the second suggestion, I thought we have lines  after the **Default** section but its right to add on top section as well in case one skips to read further.